dashboard/test/controllers/api/v1/pd/workshop_organizers_controller_test.rb

```diff
@@ -0,0 +1,44 @@
+require 'test_helper'
+
+class Api::V1::Pd::WorkshopOrganizersControllerTest < ::ActionController::TestCase
+  test 'admins can see workshop organizers' do
```

Comment:
I feel like I see this test a lot - so much so that maybe there's a way to DRY this